src/platform/src/main/java/org/geoserver/platform/RenderingEngineStatus.java

```diff
@@ -5,41 +5,16 @@
 package org.geoserver.platform;
 
 import java.util.Optional;
-import sun.java2d.pipe.RenderingEngine;
 
 /** @author Morgan Thompson - Boundless */
 public class RenderingEngineStatus implements ModuleStatus {
 
-    private static final String UNKNOWN = "unknown";
+    private static final String DEFAULT = "PLATFORM DEFAULT";
 
-    private String engine;
     private String provider;
 
-    @SuppressWarnings("unchecked")
     public RenderingEngineStatus() {
-
-        Class<RenderingEngine> renderer;
-        try {
-            renderer =
-                    (Class<RenderingEngine>)
-                            sun.java2d.pipe.RenderingEngine.getInstance().getClass();
-        } catch (Throwable e) {
-            engine = UNKNOWN;
-            provider = UNKNOWN;
-            return;
-        }
-        engine = renderer.getSimpleName();
-
-        Package pkg = renderer.getPackage();
-        if (pkg.getName().contains("marlin")) {
-            provider = "Marlin";
-        } else if (pkg.getName().contains("sun.dc")) {
-            provider = "OracleJDK";
-        } else if (pkg.getName().contains("sun.java2d")) {
-            provider = "OpenJDK";
-        } else {
-            provider = pkg.getName();
-        }
+        this.provider = System.getProperty("sun.java2d.renderer", DEFAULT);
```

Comment:
Q: So in OpenJDK 8 or Oracle JDK 8 this would just show up as empty right?

Reply:
It would show up a "PLATFORM DEFAULT"

Reply:
Yes, it would show up as PLATFORM DEFAULT with no config. Realistically this status is only going to be meaningful running anything pre JDK8 I guess.